api/admin/client_test.go

```diff
@@ -337,9 +337,10 @@ func TestGetLoggerLevel(t *testing.T) {
 				tt.loggerName,
 			)
 			require.ErrorIs(err, tt.clientErr)
-			if tt.clientErr == nil {
-				require.EqualValues(tt.serviceResponse, res)
+			if tt.clientErr != nil {
+				return
```

Comment:
I think this should be `continue` rather than return

Reply:
`return` is correct - this should not (and can not) be a `continue`